components/navigation/communityItems.js

```diff
@@ -1,8 +1,16 @@
+import IconTools from '../icons/Tools'
+import IconGithubOrganization from '../icons/GithubOrganization'
+import IconSlack from '../icons/Slack'
+import IconContributing from '../icons/Contributing'
+import IconTSC from '../icons/TSC'
+import IconMeetings from '../icons/Meetings'
+
+
 export default [
-  { text: 'Tools & Services', href: '/docs/community/tooling', description: 'Explore the tools and services our awesome community has created.' },
-  { text: 'Github Organization', href: 'https://github.com/asyncapi', target: '_blank', description: 'Want to sneak in the code? Everything we do is open-sourced in our Github organization.' },
-  { text: 'Slack Workspace', href: 'https://asyncapi.com/slack-invite', target: '_blank', description: `Need help? Want to share something? Join our Slack workspace. We're nice people :)` },
-  { text: 'Contributing', href: 'https://github.com/asyncapi?type=source#-contribute-to-asyncapi', target: '_blank', description: `We are always welcoming and looking for contributions. If you are interested check out our contribution guide.` },
-  { text: 'Technical Steering Committee', href: '/community/tsc', description: 'Get to know what is a TSC member, how you can become one, and meet our current TSC members.' },
-  { text: 'Meetings', href: '/community/meetings', description: 'See what meetings are organized under AsyncAPI umbrella and join one of them.' }
-]
\ No newline at end of file
+  { icon: IconTools, title: 'Tools & Services', href: '/docs/community/tooling', description: 'Explore the tools and services our awesome community has created.' },
+  { icon: IconGithubOrganization, title: 'Github Organization', href: 'https://github.com/asyncapi', target: '_blank', description: 'Want to sneak in the code? Everything we do is open-sourced in our Github organization.' },
+  { icon: IconSlack, title: 'Slack Workspce', href: 'https://asyncapi.com/slack-invite', target: '_blank', description: `Need help? Want to share something? Join our Slack workspace. We're nice people :)` },
```

Comment:
Correct the spelling of `Workspace` here.

Reply:
Done. Thank you.